pylsp_mypy/plugin.py

```diff
@@ -250,6 +250,7 @@ def pylsp_lint(
                     errors.strip(),
                 )
                 subprocess.run(["dmypy", "kill"], **windows_flag)
+                subprocess.run(["dmypy", "start"], **windows_flag)
```

Comment:
Nitpicking :)

```suggestion
                subprocess.run(["dmypy", "restart"], **windows_flag)
```
`dmypy --help` states
```
...
restart           Restart daemon (stop or kill followed by start)
...
```